### ISSUE TITLE
[Bug]: 'wallet must has at least one account'

### ISSUE THREAD

**Issue by allen-hsu** [MetaMask/metamask-extension#15686]
### Describe the bug

Hi, metamask team, I meet this bug on every dApp, when I click connect wallet , the metamask will not shown.
so I open the chrome console, see this error :
```
{code: 4001, message: 'wallet must has at least one account'}
```

and I try to clear the Metamask cache, browser cache/cookie/history, and reinstall the metamask.
but still can't work for me.  could you can help this ?

### Steps to reproduce

1. I don't know

### Error messages or log output

```shell
{code: 4001, message: 'wallet must has at least one account'}
```

### Version

10.18.3

### Browser

Chrome

### Operating system

MacOS

**Comment by cheshirrrcat** [comment]
I got the same issue

**Comment by Agamatus** [comment]
Did you find how to solve that?

**Comment by Agamatus** [comment]
> I got the same issue


Did you find how to solve that?

**Comment by Agamatus** [comment]
personally for me helped disabling other wallets extensions and clearing only cache. Try to experement with that. Also helps anothe browser profile or another browser

**Comment by cheshirrrcat** [comment]
> Did you find how to solve that?

Nope. I just start using another browser 😞

**Comment by Agamatus** [comment]
> > Did you find how to solve that?
> 
> Nope. I just start using another browser 😞

try to do as I said above

**Comment by CristianRicharte6** [comment]
Hi! I am going through the same error in Google Chrome when I try to connect my Metamask Wallet, I have tried from different computers, wallets and accounts and still the same error.

I have tried in Brave (Browser) and it works. Any idea how to solve it in Google Chrome?  The library I am using to talk to the blockchain is EthersJS.

Thanks in advance! 

<img width="767" alt="image" src="https://user-images.githubusercontent.com/102038261/206572275-5122ec53-8a64-4f80-ab88-e0f3a51b36cf.png">
